providers/bind/bindProvider.go

```diff
@@ -40,12 +40,14 @@ var features = providers.DocumentationNotes{
 	providers.CanUseDNAME:            providers.Can(),
 	providers.CanUseDS:               providers.Can(),
 	providers.CanUseDNSKEY:           providers.Can(),
+	providers.CanUseHTTPS:            providers.Can(),
 	providers.CanUseLOC:              providers.Can(),
 	providers.CanUseNAPTR:            providers.Can(),
 	providers.CanUsePTR:              providers.Can(),
 	providers.CanUseSOA:              providers.Can(),
 	providers.CanUseSRV:              providers.Can(),
 	providers.CanUseSSHFP:            providers.Can(),
+	providers.CanUseSVCB:             providers.Can(),
```

Comment:
@tlimoncelli, would it be an idea to call on all provider maintainers and ask them if they would like to test these two capabilities and determine whether these capabilities can be used for their provider?

Reply:
_Obviously outside the scope of this GitHub pull request._

Reply:
I can at least confirm it is working also for Cloudflare.

Reply:
Yes, once this is merged please post to the mailing list and ask maintainers to add support.